lib/swtpmtest.pm

```diff
@@ -36,8 +37,8 @@ my $guestvm_cfg = {
 };
 
 my $sample_cfg = {
-    uefi => {vm_name => 'vm-swtpm-uefi', sample_file => 'swtpm_uefi.xml'},
-    legacy => {vm_name => 'vm-swtpm-legacy', sample_file => 'swtpm_legacy.xml'},
+    uefi => {vm_name => 'vm-swtpm-uefi', sample_file => 'swtpm_uefi'},
+    legacy => {vm_name => 'vm-swtpm-legacy', sample_file => 'swtpm_legacy'},
 };
 
 # Function "start_swtpm_vm" starts a vm via libvirt "virsh" commands,
```

Comment:
Remove comma.

Reply:
FMO, it's not necessary to remove the comma here since there is a comma before here.

Reply:
Done, thanks for your comments.